src/util.ts

```diff
@@ -46,50 +46,57 @@ export const parseTypeScriptErrors = (
         line: parseInt(lineStr),
         column: parseInt(columnStr)
       }
-      const key = getSpecificErrorHash(error)
-      errors.set(key, error)
+      const key = error.file
+      const existingFile = specificErrorsMap.get(key)
+      if (existingFile) {
+        existingFile.push(error)
+        specificErrorsMap.set(key, existingFile)
```

Comment:
Since you're already mutating the underlying `existingFile` array here, this line doesn't seem necessary, but it's also fine as-is 🤷

Reply:
Nice catch - thanks!